codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -34,7 +29,11 @@ type partition = {
   id: string,
   status: status,
   latestFetchedBlock: blockNumberAndTimestamp,
-  kind: partitionKind,
+  selection: selection,
+  contractAddressMapping: ContractAddressingMap.mapping,
+  //Used to prune dynamic contract registrations in the event
+  //of a rollback.
+  dynamicContracts: array<TablesStatic.DynamicContractRegistry.t>,
```

Comment:
Can you describe why you moved these items from what was "kind"? Do these values exist for wildcard partitions?

It's not a problem I just want to understand reasoning.

Reply:
So it's not recreated and can be used as a cache key